### PR TITLE
DBAAS-530: instance page changes

### PR DESCRIPTION
**Pull request by olavtar**
Jira: https://issues.redhat.com/browse/DBAAS-530
Modified the create instance page based on the new UXD design.

Signed-off-by: Olga Lavtar <olavtar@redhat.com>

### CODE REVIEW

File: src/components/providerClusterProvisionPage.jsx [hunk 1]
```diff
@@ -105,9 +106,7 @@ const ProviderClusterProvisionPage = () => {
   const [clusterProvisionSuccess, setClusterProvisionSuccess] = React.useState(false)
   const [provisionRequestFired, setProvisionRequestFired] = React.useState(false)
   const [isDBProviderFieldValid, setIsDBProviderFieldValid] = React.useState('')
-  const [isDBProviderFieldDisabled, setIsDBProviderFieldDisabled] = React.useState(false)
```
Comment (yzhao583) [inline src/components/providerClusterProvisionPage.jsx:108]: @olavtar Why this is removed? This is required for developer create db instance.

File: src/components/providerClusterProvisionPage.jsx [hunk 2]
```diff
@@ -105,9 +106,7 @@ const ProviderClusterProvisionPage = () => {
...
   const [isInventoryFieldValid, setIsInventoryFieldValid] = React.useState('')
-  const [isInventoryFieldDisabled, setIsInventoryFieldDisabled] = React.useState(false)
```
Comment (yzhao583) [inline src/components/providerClusterProvisionPage.jsx:110]: @olavtar Same with above.

File: src/components/providerClusterProvisionPage.jsx [hunk 3]
```diff
@@ -140,7 +139,6 @@ const ProviderClusterProvisionPage = () => {
       setSelectedDBProvider(provider)
       filterInventoriesByProvider(provider)
       setIsDBProviderFieldValid(ValidatedOptions.default)
-      setIsDBProviderFieldDisabled(true)
```
Comment (yzhao583) [inline src/components/providerClusterProvisionPage.jsx:143]: @olavtar Same here

File: src/components/providerClusterProvisionPage.jsx [hunk 4]
```diff
@@ -150,7 +148,6 @@ const ProviderClusterProvisionPage = () => {
       checkInventoryStatus(inventory)
       setSelectedInventory(inventory)
       setIsInventoryFieldValid(ValidatedOptions.default)
-      setIsInventoryFieldDisabled(true)
```
Comment (yzhao583) [inline src/components/providerClusterProvisionPage.jsx:153]: @olavtar Same here

File: src/components/providerClusterProvisionPage.jsx [hunk 5]
```diff
@@ -534,7 +506,6 @@ const ProviderClusterProvisionPage = () => {
                   validated={isInventoryFieldValid}
                 >
                   <FormSelect
-                    isDisabled={isInventoryFieldDisabled}
```
Comment (yzhao583) [inline src/components/providerClusterProvisionPage.jsx:537]: @olavtar Same here

**Review by yzhao583 — CHANGES_REQUESTED** [review]
@olavtar We should not remove the disable provider account related logic, since it is required for developer to create database instance.

File: src/components/providerClusterProvisionPage.jsx (hunk 1 above)
Comment (olavtar) [reply src/components/providerClusterProvisionPage.jsx:108]: DBProviderField is completely removed from the page.

File: src/components/providerClusterProvisionPage.jsx (hunk 2 above)
Comment (olavtar) [reply src/components/providerClusterProvisionPage.jsx:110]: @yzhao583  InventoryField is Enabled in the new design, so they can change the provider account on the instance page.

Comment (yzhao583) [reply src/components/providerClusterProvisionPage.jsx:110]: @olavtar This page is used by developer as well, from the [design](https://marvelapp.com/prototype/16dd9cj9/screen/84713496), we can see that the Provider account field is disabled when developer create instance.

**Review by yzhao583 — APPROVED** [review]

**Review by yzhao583 — APPROVED** [review]

**Pull request merged**